packages/angular/projects/ui-angular/src/lib/components/authenticator/components/form-field/form-field.component.html

```diff
@@ -34,8 +34,13 @@
     [autocomplete]="formField.autocomplete"
     [type]="formField.type"
   ></amplify-text-field>
-
-  <amplify-error *ngIf="error">
-    {{ error }}
-  </amplify-error>
+  <div data-amplify-sign-up-errors *ngIf="errors?.length > 0">
+    <div
+      class="amplify-text"
+      data-variation="error"
+      *ngFor="let error of errors"
+    >
+      {{ translate(error) }}
+    </div>
```

Comment:
Won't this break the normal errors? So some errors are just red text. But the errors if you send a wrong password should be an `amplify-error` with a red box around it right? Or are those errors somewhere else?

Reply:
These are only for validation error (under each `form-field`). `remoteError`s still will be handled from each route: e.g.

https://github.com/aws-amplify/amplify-ui/blob/aad9a41ecf90a9f081c974f4e7e134a9bb9ff277/packages/angular/projects/ui-angular/src/lib/components/authenticator/components/sign-in/sign-in.component.html#L16-L18

Reply:
👍